.circleci/config.yml

```diff
@@ -277,7 +277,7 @@ jobs:
       - run:
           name: Run Small Tests
           command: |
-            SKIP_AUTO_COMPILE=true KEEP_COVER_RUNNING=1 ./tools/test.sh -p small_tests -s true -e true
+            SKIP_AUTO_COMPILE=true KEEP_COVER_RUNNING=1 REBAR_CT_EXTRA_ARGS=" -c " ./tools/test.sh -p small_tests -s true -e true
```

Comment:
do we gather cover for github actions?

Reply:
Huh, good question, no idea 🤔

Reply:
probably no, or at least not codecov service (so it does not cause interference).

I could add it though:
- it would make scripts more similar in feature set
- it would make execution a bit slower :)

I could also add an env-var COVER_ENABLED instead of that REBAR_CT_EXTRA_ARGS and maybe set it explicitly to 0 on github actions.

Reply:
done ;)
Now we just use COVER_ENABLED=false to be passed into `tools/test.sh`.
I also cleaned other env vars in circleci config.